### PR TITLE
Changes based on course evaluation

### PR DESCRIPTION
**Pull request by fasterius**
Mostly minor additions in Git/Conda (where more details was asked for), some text describing the "project" work in more detail and grammar/visual edits.

### CODE REVIEW

File: docs/git.md [hunk 1]
```diff
@@ -424,11 +425,7 @@ dependencies:
 - fastqc=0.11.6
 - sra-tools=2.8
 - snakemake-minimal=5.3.0
-<<<<<<< HEAD
```
Comment (verku) [inline docs/git.md]: In the preview here it looks as if you only deleted the info about the conflict from the conda environment file but I'm sure that is not what you intended to do.

File: docs/git.md [hunk 2]
```diff
@@ -828,6 +849,9 @@ The above command only deleted the local branch. If you want to remove the branc
 git push origin --delete test_alignment
 ```
 
+!!! tip "Tip"
```
Comment (verku) [inline docs/git.md]: Delete "Tip"?

**Comment by verku** [comment]
I didn't run the docker, singularity and project code last time (will do that at some point) so I can give you more detailed feedback later. But from my side, it looks good for merging (please note the two small comments I made)!

File: docs/git.md (hunk 1 above)
Comment (johnne) [reply docs/git.md]: Yes the conflict info should be left in there. We can however update it when the `env-updates` branch is merged.

File: docs/repres_project.md [hunk 3]
```diff
@@ -1,14 +1,13 @@
-# Using the tools to put everything together
+# Putting everything together
 
 It is time to try to setup a project from scratch and use the different
-tools that we have covered during the course together!
-
-This exercise if very open-ended and you have free hands to try out a
-bit what you want. But you should aim to use what you've learned to
-do the following:
+tools that we have covered during the course together! This exercise if very
```
Comment (johnne) [inline docs/repres_project.md]:
```suggestion
tools that we have covered during the course together! This exercise is very
```

File: docs/repres_project.md [hunk 4]
```diff
@@ -44,11 +40,13 @@ https://docs.google.com/spreadsheets/d/1mBp857raqQk32xGnQHd6Ys8oZALgf6KaFehfdwqM
...
+The goal here is to create a Snakemake workflow, which contains the following:
 
-1. has a rule that downloads the csv files (making use of a `config.yml` file to pass the URLs and file names)
-2. has a rule that cleans the files (making use of `wildcards` so that the same rule can be run on each file)
-3. The final step is to plot the student experience in some way.
+1. A rule that downloads the csv files (making use of a `config.yml` file to
+   pass the URLs and file names)
+2. A rule that cleans the files (making use of `wildcards` so that the same
+   rule can be run on each file)
+3. A final step that cab plot the student experience in some way.
```
Comment (johnne) [inline docs/repres_project.md]:
```suggestion
3. A final step that can plot the student experience in some way.
```

File: docs/git.md (hunk 1 above)
Comment (fasterius) [reply docs/git.md]: Okay, so no changes on this PR regarding this comment?

File: docs/git.md (hunk 2 above)
Comment (fasterius) [reply docs/git.md]: I was unsure how the formatting was done with admonitions, as this one (and another one) was not rendering properly, so I added this. Maybe it's unnecessary, though? Just thought that it was weird that it wasn't rendering.

File: docs/git.md (hunk 1 above)
Comment (johnne) [reply docs/git.md]: Right, we'll fix it when merging `env-updates` which will cause conflicts anyway.

**Comment by johnne** [comment]
I'm also fine with merging!

**Pull request merged**